src/components/Banner/Banner.tsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { isNotUndefined } from 'ramda-adjunct';
+
+import { BannerProps } from './Banner.types';
+import { BannerVariants } from './Banner.enums';
+import { Icon } from '../Icon';
+import { Button } from '../Button';
+import { Inline, Padbox } from '../layout';
+import colors from '../../theme/colors';
+import { IconTypes, SSCIconNames } from '../../theme/icons/icons.enums';
+import { getColor, pxToRem } from '../../utils';
+
+const colorVariants = {
+  [BannerVariants.info]: colors.radiantBlueberry,
+  [BannerVariants.warn]: colors.pumpkin,
+  [BannerVariants.error]: colors.strawberry,
+};
+
+const iconVariants = {
+  [BannerVariants.info]: SSCIconNames.infoCircle,
+  [BannerVariants.warn]: SSCIconNames.exclamationTriangle,
+  [BannerVariants.error]: SSCIconNames.ban,
+};
+
+const bgVariants = {
+  [BannerVariants.info]: { r: 235, g: 244, b: 252, a: 0.5 },
+  [BannerVariants.warn]: { r: 240, g: 143, b: 0, a: 0.05 },
+  [BannerVariants.error]: { r: 241, g: 67, b: 28, a: 0.05 },
+};
+
+const StyledInline = styled(Inline)<Pick<BannerProps, 'variant'>>`
+  border: ${({ variant }) => `1px solid ${colorVariants[variant]}`};
+  background: ${({ variant }) =>
+    `rgba(${bgVariants[variant].r}, ${bgVariants[variant].g}, ${bgVariants[variant].b}, ${bgVariants[variant].a})`};
+`;
+
+const IconContainer = styled.div<Pick<BannerProps, 'variant'>>`
+  background-color: ${({ variant }) => colorVariants[variant]};
+  height: 100%;
+  display: flex;
+  align-items: center;
+`;
+
+const StyledIcon = styled(Icon)`
+  color: ${colors.graphite5H};
+`;
+
+const CloseButton = styled.button`
```

Comment:
Could you please add a TODO comment here to take a look at other close buttons we might have in DS to extract that and unify it